.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What's the purpose of this file?

Reply:
it was added automatically by Eclipse .. it is not necessary

Reply:
IMO better to just remove it then, consider adding this to gitignore